css-layout-classes/styles.css

```diff
@@ -0,0 +1,32 @@
+.container {
+  margin: auto;
+  width: 75%;
+}
+
+.row {
+  display: flex;
+}
+
+.col-3 {
+  flex-basis: calc (100% / 3);
+}
+
+.col-4 {
+  flex-basis: 25%;
+}
+
+.justify-center {
+  justify-content: center;
+}
+
+.row-3 {
+  width: 95%;
+}
+
+.col-two-thirds {
+  flex-basis: calc(200% / 3);
+}
+
+.col-mario {
+  flex-basis: calc(200% / 3);
+}
```

Comment:
This is the same styling as the above `col-two-thirds` class, you should be able to reuse that.